package.json

```diff
@@ -220,9 +220,8 @@
     "yaml": "^1.10.0"
   },
   "scripts": {
-    "prepublish": "gulp post-npm-install",
+    "prepack": "gulp post-npm-install",
     "postpublish": "bash -c \"if [ -z \"$GITHUB_ACTION\" ]; then git tag -a ${npm_package_version} -m \"${npm_package_version}\" && git push origin ${npm_package_version}; fi \"",
-    "postinstall": "gulp post-npm-install",
```

Comment:
I think `gulp post-npm-install` is always run after building, so removed the hook since the warning box in the documentation is quite strongly worded against this hook: https://yarnpkg.com/advanced/lifecycle-scripts#postinstall